src/lib/services/api-token-service.ts

```diff
@@ -111,7 +116,12 @@ export class ApiTokenService {
     }
 
     public async getAllActiveTokens(): Promise<IApiToken[]> {
-        return this.store.getAllActive();
+        if (!this.initialized) {
+            // unlikely this will happen but nice to have a fail safe
+            this.logger.info('Fetching active tokens before initialized');
+            await this.fetchActiveTokens();
+        }
+        return this.activeTokens;
```

Comment:
This method now returns from memory, but if not yet initialized will attempt to load the tokens. This is just a safeguard